package a4.io;

message Cutflow
{
    optional string name = 1;
    repeated double counts_double = 2 [packed=true];
    repeated string counts_double_names = 3;
    repeated double weights_squared = 14 [packed=true];
}
